src/main/java/ChatRequest.java

```diff
@@ -24,4 +24,47 @@ public boolean newMethod(String st1 ,String st2){
             }
             return false;
         }
+        public String chat(String question){
+            /* Initialize Variable */
+            PromptTemplate promptTemplate = null;
+            try {
+                /* Create a prompt template using the question */
+                promptTemplate = new PromptTemplate(question);
+            }catch(Exception e) {
```

Comment:
Catching a generic Exception is a bad practice as it can hide specific issues. It's better to catch specific exceptions that you expect. (Severity: High)